blocks_vertical/data.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+goog.provide('Blockly.Blocks.data');
+
+goog.require('Blockly.Blocks');
+
+goog.require('Blockly.Colours');
+
+goog.require('Blockly.constants');
+
+Blockly.Blocks['data_variablemenu'] = {
+  /**
+   * Variable menu.
+   * @this Blockly.Block
+   */
+  init: function() {
+    this.jsonInit(
+      {
+        "message0": "%1",
+        "args0": [
+          {
+            "type": "field_variable",
+            "name": "VARIABLE",
+            "options": [
+              ['v', 'V'],
+              ['v2', 'V2'],
+              ['v3', 'V3'],
+              ['v4', 'V4'],
+              ['v5', 'V5'],
+              ['v6', 'V6'],
+              ['v7', 'V7'],			  
+            ]
+          }
+        ],
+        "inputsInline": true,
+        "output": "String",
+        "colour": Blockly.Colours.data.secondary,
+        "colourSecondary": Blockly.Colours.data.secondary,
+        "colourTertiary": Blockly.Colours.data.tertiary,
+        "outputShape": Blockly.OUTPUT_SHAPE_ROUND
+      });
+  }
+};
+
+Blockly.Blocks['data_variable'] = {
+  /**
+   * Block of Variables
+   * @this Blockly.Block
+   */
+  init: function() {
+    this.jsonInit({
+      "message0": "%1",
+        "args0": [
+          {
+            "type": "input_value",
+            "name": "VARIABLE"
+          }
+        ],  
+      "colour": Blockly.Colours.data.primary,
+      "colourSecondary": Blockly.Colours.data.secondary,
+      "colourTertiary": Blockly.Colours.data.tertiary,
+      "output": "Number",

```

Comment:
(A *little* off-topic, could fit well in a new issue, but..)

Does this mean that a variable containing the value `true` will now be the same (when called) as a variable containing the value `"true"`?

For example, in this demo:

![Demo](https://cloud.githubusercontent.com/assets/9948030/16667826/ccb95c78-4464-11e6-945c-f69c9f040d03.png)

`0 + "true"` is `0` and `0 + true` is `1`. Assuming to-number convertion stays the same in Scratch 3.0, and variables *always* output strings – because their `output` is `"String"` – can we expect `0 + (var whose value is: true)` to be `0`?